gaffer-as-a-service/gaas-rest/deploy/helm/templates/deployment.yaml

```diff
@@ -42,3 +42,31 @@ spec:
               limits:
                 memory: "1Gi"
                 cpu: "1000m"
+          - image: dockerhub/gs-sc-img
```

Comment:
What are these images?

Reply:
These are just placeholder images but they are for the sidecar and eureka server to be deployed alongside rest. To make this more manageable I have parameterised this so it can be done from values.yaml.